internal/buffer/eventhandler.go

```diff
@@ -247,6 +247,15 @@ func (eh *EventHandler) Execute(t *TextEvent) {
 	}
 
 	ExecuteTextEvent(t, eh.buf)
+
+	b, err := config.RunPluginFnBool("onExecuteTextEvent", luar.New(ulua.L, eh.buf), luar.New(ulua.L, t))
```

Comment:
Why such a non-intuitive name. Why not `onAfterTextEvent`, or better just `onTextEvent`?

(BTW `onBeforeTextEvent` was an unfortunate name, I think. `preTextEvent` would be better, it would be consistent with other callbacks: `preRune`, `preAction`.)